src/composables/LetoModelizerApi.js

```diff
@@ -38,3 +38,21 @@ export async function getUserRoles(userId, sessionToken) {
 
   return api.get(`/api/roles?${queryParameters}`, headers);
 }
+
+/**
+ * Get all libraries.
+ * @param {string} sessionToken - The current user's session token.
+ * @returns {Promise<object[]>} Return an array of libraries.
+ */
+export async function getLibraries(sessionToken) {
```

Comment:
you can remove `sessionToken` from parameters and put use directly method to get it in the function.

Reply:
done